TeensyDmx.cpp

```diff
@@ -465,121 +453,118 @@ void TeensyDmx::rdmUniqueBranch(const unsigned long timingStart, struct RDMDATA*
             *m_redePin = 1;
         }
         m_dmxBufferIndex = 0;
-        m_uart.begin(RDM_BREAKSPEED, BREAKFORMAT);
-        m_uart.write(0);
-        m_uart.flush();
+        // No break for DUB
         m_uart.begin(DMXSPEED, DMXFORMAT);
         for (uint16_t i = 0; i < sizeof(struct DISCOVERYMSG); ++i) {
-            m_uart.write(m_activeBuffer[i]);
+            m_uart.write(m_rdmBuffer.buffer[i]);
             m_uart.flush();
         }
         startReceive();
     }
 }
 
-void TeensyDmx::rdmUnmute(const unsigned long timingStart, struct RDMDATA* rdm)
+uint16_t TeensyDmx::rdmUnmute(struct RDMDATA* rdm)
 {
     if (rdm->DataLength == 0) {
         m_rdmMute = false;
         // Control field
         rdm->Data[0] = 0;
         rdm->Data[1] = 0;
         rdm->DataLength = 2;
-        respondMessage(timingStart, NACK_WAS_ACK);
+        return NACK_WAS_ACK;
+    } else {
+        return E120_NR_FORMAT_ERROR;;
```

Comment:
Superflous semi-colons here and on each of the other functions.

Reply:
Done